docs/guides/installation.md

```diff
@@ -24,10 +24,174 @@ Installation requires the following:
 Clone the [ZARP workflow repository][zarp] with:
```

Comment:
In row 18 above, there is a **TODO** left (good reminder why it's not a good idea to leave TODOs in the code)

Reply:
Good catch. Fixed.